locales/README.es_ES.md

```diff
@@ -18,13 +18,13 @@ Rocky Linux es un Sistema Operativo empresarial comunitario diseñado para ser 1
 
 > **P:** ¿A qué te refieres con, "Centos ha cambiado de dirección"?
 
-El proyecto CentOS [recientemente anunció](https://blog.centos.org/2020/12/future-is-centos-stream/) un cambio en su estrategia para CentOS. Mientras previamente CentOS existía como un build *downstream* de su proveedor upstream (recibía parches y actualizaciones luego del proveedor), ahora cambiará a ser un build *upstream* (testeando parches y actualziaciones *antes* de su inclusión en el proveedor upstream).
+El proyecto CentOS [recientemente anunció](https://blog.centos.org/2020/12/future-is-centos-stream/) un cambio en su estrategia para CentOS. Mientras previamente CentOS existía como un build *downstream* de su proveedor upstream (recibía parches y actualizaciones luego del proveedor), ahora cambiará a ser un build *upstream* (testeando parches y actualizaciones *antes* de su inclusión en el proveedor upstream).
 
-En adición a eso, el soporte para CentOS Linux 8 se acortó, del 31 de Mayo de 2029 al 31 de Diciembre de 2021.
+En adición a eso, el soporte para CentOS Linux 8 se acortó, del 31 de mayo de 2029 al 31 de diciembre de 2021.
```

Comment:
Maybe better "Añadido a eso" or "Además de eso"

Reply:
quizás esto seria bueno para otro parche.
(solo arreglando errores de convenciones con respecto a fechas en este parrafo)

Una cosa a la vez.

Reply:
"Añadido a eso" or "Además de eso" is much better than "En adición a eso"... "Adicionalmente" may be the best?